js/src/ui/index.js

```diff
@@ -14,118 +14,47 @@
-import Certifications from './Certifications';
-import ConfirmDialog from './ConfirmDialog';
-import Container, { Title as ContainerTitle } from './Container';
-import ContextProvider from './ContextProvider';
-import CopyToClipboard from './CopyToClipboard';
-import CurrencySymbol from './CurrencySymbol';
-import DappCard from './DappCard';
-import DappIcon from './DappIcon';
-import Editor from './Editor';
-import Errors from './Errors';
-import Features, { FEATURES, FeaturesStore } from './Features';
-import Form, { AddressSelect, DappUrlInput, FormWrap, Input, InputAddress, InputAddressSelect, InputChip, InputDate, InputInline, InputTime, Label, RadioButtons, Select, TypedInput } from './Form';
-import GasPriceEditor from './GasPriceEditor';
-import GasPriceSelector from './GasPriceSelector';
-import Icons from './Icons';
-import IdentityIcon from './IdentityIcon';
-import IdentityName from './IdentityName';
-import LanguageSelector from './LanguageSelector';
-import Loading from './Loading';
-import MethodDecoding from './MethodDecoding';
-import Modal, { Busy as BusyStep, Completed as CompletedStep } from './Modal';
-import muiTheme from './Theme';
-import Page from './Page';
-import ParityBackground from './ParityBackground';
-import PasswordStrength from './Form/PasswordStrength';
-import Portal from './Portal';
-import QrCode from './QrCode';
-import SectionList from './SectionList';
-import ShortenedHash from './ShortenedHash';
-import SignerIcon from './SignerIcon';
-import Tags from './Tags';
-import Title from './Title';
-import Tooltips, { Tooltip } from './Tooltips';
-import TxHash from './TxHash';
-import TxList from './TxList';
-import Warning from './Warning';
-
-export {
-  AccountCard,
-  Actionbar,
-  ActionbarExport,
-  ActionbarImport,
-  ActionbarSearch,
-  ActionbarSort,
-  AddressSelect,
-  Badge,
-  Balance,
-  BlockStatus,
-  Button,
-  Certifications,
-  ConfirmDialog,
-  Container,
-  ContainerTitle,
-  ContextProvider,
-  CopyToClipboard,
-  CurrencySymbol,
-  DappCard,
-  DappIcon,
-  DappUrlInput,
-  Editor,
-  Errors,
-  FEATURES,
-  Features,
-  FeaturesStore,
-  Form,
-  FormWrap,
-  GasPriceEditor,
-  GasPriceSelector,
-  Icons,
-  Input,
-  InputAddress,
-  InputAddressSelect,
-  InputChip,
-  InputDate,
-  InputInline,
-  InputTime,
-  IdentityIcon,
-  IdentityName,
-  Label,
-  LanguageSelector,
-  Loading,
-  MethodDecoding,
-  Modal,
-  BusyStep,
-  CompletedStep,
-  muiTheme,
-  Page,
-  ParityBackground,
-  PasswordStrength,
-  Portal,
-  QrCode,
-  RadioButtons,
-  Select,
-  ShortenedHash,
-  SectionList,
-  SignerIcon,
-  Tags,
-  Title,
-  Tooltip,
-  Tooltips,
-  TxHash,
-  TxList,
-  TypedInput,
-  Warning
-};
+export AccountCard from './AccountCard';
+export Actionbar from './Actionbar';
+export ActionbarExport from './Actionbar/Export';
+export ActionbarImport from './Actionbar/Import';
+export ActionbarSearch from './Actionbar/Search';
+export ActionbarSort from './Actionbar/Sort';
```

Comment:
Aligning with the exports as done in `Form` & `Container` would actually be quire good -

```
export Actionbar, { Export as ActionbarExport, ...} from './Actionbar;
```

Alternatively since these are sub components, making them static on `Actionbar`, only exporting it and then using `Actionbar.Export` could also work.